kqueen_ui/blueprints/ui/templates/ui/partial/_cluster_row.html

```diff
@@ -38,7 +38,7 @@
             {% endif %}
           </td>
           <td class="provisioner_column">{{ cluster.provisioner.name }}</td>
-          <td class="created_column">{{ cluster.created_at }}</td>
+          <td class="created_column">js-replace-datetime({{ cluster.created_at }})</td>
```

Comment:
Now it works as follows:
1. The page is loaded with empty table content (there is a loading animation).
2. JS sends a query to get this html (as a string).
3. All `js-replace-datetime` occurences in this string are replaced with formatted date.
4. The resulting string replaces the table.

Your suggestion:
1. The page is loaded.
2. JS sends a query.
3. JS replaces the table with the response string.
4. We find all elements with this datetime data (or with a specific class) and replace their content in DOM.

I think that the performance of the first approach is better, and it inserts already formatted date instead of replacing after rendering. 
What do you think?

Reply:
you still need to find elements either by text or by class. if we'll use class, user will see just time in (in case of large tables) and all replacements will be in one place

Reply:
> you still need to find elements either by text or by class
Well, I just replace a text in a string. And it happens before elements are displayed on a page. So users see proper time from the very beginning.